### ISSUE TITLE
Looking for a maintainer for redux-cycles

### ISSUE THREAD

**Issue by Widdershin** [cyclejs-community/cyclejs-community#18]
Hi @cyclejs-community/contributors 😄 

I hope y'all are doing well. It's been a while since we've had any community-wide communication go out, but in the meantime things have been ticking along.

I'm reaching out to you because we're looking for someone to step up as a maintainer for [redux-cycles](https://github.com/cyclejs-community/redux-cycles/).

For those who aren't familiar, `redux-cycles` is a cool piece of technology that acts as a bridge between `redux` and `cycle`, and in my opinion is one of the most elegant solutions for managing asynchronous effects for apps that use `redux`. It's also one of the most starred pieces of tech in the Cycle.js ecosystem!

`redux-cycles` was originally created by @nickbalestra and @lmatteis, who did a fantastic job launching the project, but it currently has nobody actively working on it.

A major part of the reason @TylorS and I created this community is because we believe that contributors should be able to work on a project without feeling an obligation to stick around forever, or guilt if they need to walk away. We want working on open source Cycle projects to be a great experience, and in order to do that we need to have each others backs.

To that end, I'm hoping to find one or several people who are interested in doing a bit of maintenance work on `redux-cycles`. There are several low hanging fruit issues that would go a long way, like updating the dependencies, updating the README and triaging the existing issues.

If you want to be more involved in open source or the Cycle.js community, this is a great opportunity. By putting your hand up, there's no obligation to have a long term commitment to working on the project. Any work that you can find time to do is greatly valued.

I hope that someone is willing to step up and do a little bit of work here, as it would be a shame for such a great project to be deprecated when it just needs a little bit of love.

I'm happy to provide guidance and support for anyone who wants to do some maintenance. I'll give you push access to the repo, arrange for you to have publish access on npm, and field any questions you have. If you're keen, please comment in here or get in touch with me on Gitter. Also feel free to post any questions you have in here!

**Comment by artfuldev** [comment]
✋ I'd like to put my hand up for this. I do not have a lot of time in a realistic sense, but whatever little I have I'd be happy to contribute towards this.

**Comment by nickbalestra** [comment]
Thank you @Widdershin for helping with this! And thanks @artfuldev for stepping up, really really appreciated ❤️❤️❤️! 
I'll try to share this post too to see if i can help get more people interested in contributing toward this.

**Comment by cjduncana** [comment]
✋

**Comment by cjduncana** [comment]
I can't get the example folder to run, the tests are breaking, and the TypeScript typings are outdated. Is there anywhere where you would like me to focus first?

**Comment by Widdershin** [comment]
Whatever seems easiest first would make sense to me, those are all good things to fix.

I’m out of town for a few days but once I’m back I’ll give you both push access to the repo. If you post your npm usernames in here we can get you publish rights as well.

It would also be good to have a high level issue with a checklist of what needs to be done to get the project back to a happy state, so if you’re interested in that it would be a good place to start.

**Comment by cjduncana** [comment]
This is my NPM account: https://www.npmjs.com/~cjduncana

**Comment by Widdershin** [comment]
@artfuldev and @cjduncana I've added you as contributors to the `redux-cycles` repository.

Personally I have no qualms with y'all pushing master with fixes, but feel free to do pull requests or whatever works for you. @lmatteis can you please give cjduncana npm publish rights?

Thanks again for stepping up, any work you find time for is much appreciated 😄

**Comment by cjduncana** [comment]
@Widdershin Thanks for the invite. I hope that I can provide useful tools to the community, and keep this great project going.

**Comment by artfuldev** [comment]
Mine is https://www.npmjs.com/~artfuldev

**Comment by artfuldev** [comment]
I'm more comfortable with PRs and discussions around them. But whatever works!

**Comment by cjduncana** [comment]
I agree. I also prefer PRs and discussion. Currently, I'm trying to update the dependencies and figure out how to get the tests and examples going again.

@artfuldev, have you been able to get those working with the project as is?

**Comment by lmatteis** [comment]
I've invited both artfuldev and cjduncana to the project on NPM as maintainers:

![screen shot 2018-11-21 at 13 48 53](https://user-images.githubusercontent.com/535442/48842323-41390a00-ed94-11e8-931b-ba510f6a9d18.png)

**Comment by artfuldev** [comment]
> @artfuldev, have you been able to get those working with the project as is?

@cjduncana Hoping to get started with this over the weekend. Haven't had much time lately.
